docs/shell-tips.md

```diff
@@ -164,10 +194,134 @@ have the newline character, and thus the `\<newline>`.
 The newline character is invisible (we only see a line break), but it's still
 there!
 
+## Arguments
+
+Most of the commands need additional data to work, like a path or a file. That
+extra information is called an **argument**, and it's specified after the name
+of the command, like this:
+
+```
+$ cd /home/john/notes
+```
+
+Here, the main command is `cd`, and the first (and only) argument is
+`/home/john/notes`.
+
+In each command the arguments are specified in different ways, and have
+different meanings.
+
+Sometimes, a command can accept multiple arguments. In those cases, dashes are
+used to indicate the different arguments. Here's another example:
+
+```
+$ nano -C /home/john/backups --mouse todo.txt
+```
+
+In this case, we're saying: "`nano`, open the file `todo.txt`, enabling mouse
+support, and saving the backup files to `/home/john/backups`". Don't worry, you
+don't have to memorize the meaning of all the arguments for every single
+command. There are [tools](#understanding-commands) that help you with that
+:wink:.
```

Comment:
I think when we're using keywords, we want to first explain and then use them - it's just easier on the reader. That's why I'd think we should do the split first.